src/screens/WalletScreen/Content/Content.tsx

```diff
@@ -18,29 +20,70 @@ export const Content: React.FC<ContentProps> = ({
 	setAddFundsVisible,
 	setSendModalOpen
 }) => {
+	const [txs, setTxs] = useState<ZapperTransaction[]>([]);
```

Comment:
you don't need a duplicated state for the pending transactions. it can use the pending transactions we already have.